trade-finance-manager-ui/templates/case/underwriting/managers-decision/edit-managers-decision.njk

```diff
@@ -6,6 +6,8 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle %}Underwriter manager’s decision{% endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.